app/controllers/api/v1/habits_controller.rb

```diff
@@ -52,4 +74,16 @@ def update_info(habit, info)
     end
     habit
   end
+
+  def schedules_saved?(schedules)
```

Comment:
Unused method argument - `schedules`. If it's necessary, use `_` or `_schedules` as an argument name to indicate that it won't be used. You can also write as `schedules_saved?(*)` if you want the method to accept any arguments but don't care about them.